### DISCUSSION TITLE
Renew TSA/Signserver Workers

### DISCUSSION THREAD

**Discussion by MrWildanMD** [Keyfactor/ejbca-ce#448] [Q&A] [answered]
Hello @primetomas how can i renew the signserver workers? it says the signer is expired, thanks

**Comment by primetomas** [comment, marked as answer]
This question should be asked in https://github.com/Keyfactor/signserver-ce/

Cheers,
Tomas

**Reply by MrWildanMD** [reply]
ah alright, moving discussion to signserver repo
